frontends/web/src/components/forms/select.tsx

```diff
@@ -41,7 +41,7 @@ export const Select = ({
       <select id={id} {...props}>
         {options.map(({ value, text, disabled = false }) => (
           <option
-            key={`${value}`}
+            key={String(value)}
```

Comment:
anything other than string and number typescript now complains about (with the new eslint rule in this PR) so here without converting to string somehow i.e. `String()` this will error with:

```ts
Type 'string | number | readonly string[] | undefined' is not assignable to type 'Key | null | undefined'.
  Type 'readonly string[]' is not assignable to type 'Key | null | undefined'.ts(2322)
index.d.ts(261, 9): The expected type comes from property 'key' which is declared here on type 'DetailedHTMLProps<OptionHTMLAttributes<HTMLOptionElement>, HTMLOptionElement>'
(property) Attributes.key?: Key | null | undefined
```

value comes from intrinsic element option and is `value?: string | readonly string[] | number | undefined;`

```ts
    interface OptionHTMLAttributes<T> extends HTMLAttributes<T> {
        disabled?: boolean | undefined;
        label?: string | undefined;
        selected?: boolean | undefined;
        value?: string | readonly string[] | number | undefined;
    }
  ```